tests/magick-image/motion-blur.spec.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright Dirk Lemstra https://github.com/dlemstra/magick-wasm.
+// Licensed under the Apache License, Version 2.0.
+
+import { IMagickImage, MagickImage } from '../../src/magick-image';
+
+let image: IMagickImage;
```

Comment:
You don't need this anymore. You can use `TestImages.Builtin.logo.use((image) => {` in the unit test instead.